config/Cargo.toml

```diff
@@ -10,7 +10,6 @@ publish = false
 edition = "2018"
 
 [dependencies]
-anyhow = "1.0.31"
```

Comment:
bold

Reply:
but yeah IIUC a library should use thiserror

Reply:
yeah, not sure if you read the commit or description, but anyhow is really awesome for prototyping and not falling back on expect / panic / unwrap, but then you end up in a world of hurt because it is too easy to use and hard to get right.